src/components/chats/ChatItem/ChatItem.tsx

```diff
@@ -0,0 +1,219 @@
+import AddressAvatar from '@/components/AddressAvatar'
```

Comment:
Actually this is one thing that I do in many places in this repo,
so I made a folder, in this case 
ChatItem/
  ...Supporting1.tsx
  ...Supporting2.tsx
  ChatItem.tsx
  index.tsx
 
which consists of the main export (index.tsx) which exports the main file, which is the file with the same name as the folder.

I do this so it will still be easy to navigate with `cmd+p` file search instead of putting the code directly inside `index.tsx`.